dace/transformation/dataflow/map_expansion.py

```diff
@@ -47,7 +47,7 @@ def apply(self, graph: dace.SDFGState, sdfg: dace.SDFG):
         new_maps = [
             nodes.Map(current_map.label + '_' + str(param), [param],
                       subsets.Range([param_range]),
-                      schedule=dtypes.ScheduleType.Sequential)
+                      schedule=current_map.schedule)
```

Comment:
will not work for GPU maps.

Reply:
Could you elaborate why do you think this would not work in GPU maps? The reason why I put this in is that I had maps expanded which were not sequential and suddenly one of the maps was sequential, which goes against my understanding of what MapExpansion should do. Though when I disable this change some (maybe all, didn't check all) of the failing tests pass again.